packages/opentelemetry-node-tracer/src/NodeTracer.ts

```diff
@@ -31,4 +31,15 @@ export class NodeTracer extends BasicTracer {
 
     // @todo: Integrate Plugin Loader (pull/126).
   }
+
+  /**
+   * Binds the trace context to the given event emitter.
+   * This is necessary in order to create child spans correctly in event
+   * handlers.
+   * @param emitter An event emitter whose handlers should have
+   *     the trace context binded to them.
+   */
+  wrapEmitter(emitter: NodeJS.EventEmitter, scope?: unknown): void {
+    this._scopeManager.bind(emitter, scope);
```

Comment:
We should not have specific method for a object type there since the actual handling is done inside the `ah-scope-manager`.

I would prefer having a getter for the scopeManager on the tracer or directly the `bind` method from it exposed.

Reply:
Thanks @vmarchaud 
I changed the code, let me know if this is what you looking for